docs/index.asciidoc

```diff
@@ -102,10 +102,26 @@ include::static/upgrading.asciidoc[]
 :edit_url!:
 include::static/configuration.asciidoc[]
 
+
+// Securing Logstash
```

Comment:
Rather than having separate pages, is it possible to have all of the information on a single page so that users can work their way through all of the steps? That's the thinking that drove the Elasticsearch security rewrite and provided more of a tutorial-style workflow.